internal/npm_install/generate_build_file.ts

```diff
@@ -76,42 +75,58 @@ package(default_visibility = ["${visibility}"])
 }
 
 if (require.main === module) {
-  main();
```

Comment:
There's a lot of random missing semicolons throughout the file, this example creating an unnecessary diff...

Reply:
Wow, there were more of those then I realized... it wasn't just your PR lol